src/main/resources/templates/components/errorNotification.mustache

```diff
@@ -0,0 +1,8 @@
+<div class="error_wrapper">
+    <img src="https://uploads-ssl.webflow.com/617fbee004d9ae0e43297700/6347f081beb57e1344756db2_img_icon_error_filler_red.svg" loading="lazy" alt="" class="error_icon">
+    <div>
+        <div class="error_title">Es ist ein Fehler aufgetreten.</div>
+        <div>Stellen Sie sicher, dass die BundesIdent App installiert ist und versuchen Sie es erneut.</div>
```

Comment:
Since we now store all the plain text of the widget in our `application.yaml` we should be consistent and also move this text there.